integration/service/create_test.go

```diff
@@ -123,6 +125,34 @@ func TestCreateServiceMultipleTimes(t *testing.T) {
 	poll.WaitOn(t, networkIsRemoved(client, overlayID), poll.WithTimeout(1*time.Minute), poll.WithDelay(10*time.Second))
 }
 
+func TestCreateServiceConflict(t *testing.T) {
+	skip.If(t, testEnv.DaemonInfo.OSType == "windows")
```

Comment:
Why skip?

Reply:
Setting up a swarm in tests is currently not working, that's why all these tests are currently skipped on Windows; see https://github.com/moby/moby/blob/5d82d77a6df0c95bb5986ace1255de424c700b33/integration/internal/swarm/service.go#L50-L61